js/web/lib/wasm/wasm-factory.ts

```diff
@@ -71,11 +77,14 @@ const isSimdSupported = (): boolean => {
   }
 };
 
-const getWasmFileName = (useSimd: boolean, useThreads: boolean) => {
-  if (useThreads) {
-    return useSimd ? 'ort-wasm-simd-threaded.wasm' : 'ort-wasm-threaded.wasm';
+const getWasmFileName = (useSimd: boolean, useThreads: boolean, useTraining: boolean) => {
```

Comment:
don't pass `useTraining`. use `BUILD_DEFS.ENABLE_TRAINING` and consider it as build time const

Reply:
oops that would make much more sense; updated